react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js

```diff
@@ -104,6 +104,7 @@ class ChromeExtensionBanner extends PureComponent<Props, State> {
             shouldShow: false
         };
 
+        this.isEdge = /Edg(e)?/.test(navigator.userAgent);
```

Comment:
Don't we have a helper for this in LJM's browser capabilities object?

Reply:
it only detects cromium based, not specifically edge